src/metamath.c

```diff
@@ -6313,11 +6313,15 @@ void command(int argc, char *argv[]) {
     }
 
     if (cmdMatches("VERIFY PROOF")) {
-      if (switchPos("SYNTAX_ONLY")) {
-        verifyProofs(g_fullArg[2],0); // Parse only
-      } else {
-        verifyProofs(g_fullArg[2],1); // Parse and verify
-      }
+      i = (switchPos("SYNTAX_ONLY") != 0) ? 0 : 1;
```

Comment:
Here and below, isn't our convention to write conditions as `(switchPos("SYNTAX_ONLY"))` instead of `(switchPos("SYNTAX_ONLY") != 0)` ?

Reply:
Yes, you are right.  I have changed the code accordingly.